### DISCUSSION TITLE
is Supabase tables have function like excel tables ?

### DISCUSSION THREAD

**Discussion by Othman-Sami** [supabase/supabase#15636] [Questions]
What I want to ask its I have table with some columns
one of it have two option ( active , not active )
other column have date  

can I make the Supabase server make automatic if the time out of the date make the other column change from active to not active

**Comment by GaryAustin1** [comment]
Postgres, the database Supabase uses, has a concept of generated columns.  You can have a column that is generated based on the values of other columns in the row.  This has to been done with SQL though versus the UI.  https://www.postgresql.org/docs/current/ddl-generated-columns.html
